src/unix/fs.c

```diff
@@ -1680,6 +1680,22 @@ static ssize_t uv__fs_write_all(uv_fs_t* req) {
 }
 
 
+#ifdef __MVS__
+static ssize_t uv__fs_access(uv_fs_t* req) {
+  /* On z/OS, F_OK is 8 instead of 0, so separate the access checks here. */
```

Comment:
It's not immediately clear (at least to me) just from reading this comment why `F_OK` being 8 requires special treatment. The docs I found say:

> If you are using F_OK to test for the file's existence, you cannot use OR with any of the other symbols.

Maybe the comment could just say "`F_OK` cannot be OR'ed with other access modes on z/OS."

Reply:
`F_OK` not being allowed to be OR'ed with other access modes does not appear to be a z/OS specific limitation.

e.g. https://linux.die.net/man/2/access
> The mode specifies the accessibility check(s) to be performed, and is either the value F_OK, or a mask consisting of the bitwise OR of one or more of R_OK, W_OK, and X_OK.

and https://pubs.opengroup.org/onlinepubs/9699919799/functions/access.html
> The value of amode is either the bitwise-inclusive OR of the access permissions to be checked (R_OK, W_OK, X_OK) or the existence test (F_OK).

Reply:
I should have checked the docs for other platforms 🤦 ... in that case, do we even need to do anything here?